ArcadiaParties.API/Startup.cs

```diff
@@ -36,8 +37,6 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
-            app.UseHttpsRedirection();
```

Comment:
Why?

Reply:
I see this when navigate to http://...
![image](https://user-images.githubusercontent.com/55158384/74956907-d9022980-5417-11ea-9903-a4ee1b6ea733.png)

Reply:
OK, let's remove it.